frame/base/bli_cpuid.c

```diff
@@ -1061,27 +1076,54 @@ int vpu_count( void )
 
 #elif defined(__aarch64__)
 
+#if __linux__
```

Comment:
OK, I'm not going to mark all of these.... this is great stuff but should be in a separate PR.